module/bdev/raid/bdev_raid.h

```diff
@@ -43,6 +43,24 @@ enum raid_bdev_state {
 	RAID_BDEV_STATE_MAX
 };
 
+/*
+ * The state of a base bdev
+ * When a bdev remove event or a base bdev fail message arrive to the raid, if delta map is enabled,
+ * the base bdev enter in faulty state for a defined timeout: the base bdev is removed from the
+ * raid but a delta bitmap is created and updated by every I/O write operations performed over the raid.
+ * Expired the timeout, the delta bitmap is deleted and the base bdev is removed from the faulty state,
+ * so no sign of this base bdev will remain in the raid.
+ */
+enum base_bdev_state {
+	BASE_BDEV_STATE_NONE,
```

Comment:
Does `BASE_BDEV_STATE_NONE` mean the base bdev is healthy and connected?

Reply:
Yes, put a comment